simplq/src/main/resources/db/changeset.log.xml

```diff
@@ -78,6 +78,14 @@
 
 		</addColumn>
 	</changeSet>
-	
+
```

Comment:
Can you add a short comment as to why we had to make this change?

Reply:
Done